src/settings.js

```diff
@@ -2027,6 +2027,11 @@ var TRUSTED_TYPES = false;
 // settings is *only* needed when also explicitly targeting older browsers.
 var POLYFILL = true;
 
+// Embind specific: If enabled, identifiers are parsed using 'UTF8ToString',
+// allowing the use of Unicode characters. However, there is a slight overhead 
+// during module loading compared to using 'readLatin1String'.
+var ALLOW_UNICODE_IDENTIFIER = false;
```

Comment:
Let's move this under `EMBIND_STD_STRING_IS_UTF8` in the file and prefix the name e.g. `EMBIND_ALLOW_UNICODE_IDENTIFIER`.